test/unit/mdc-textfield/mdc-textfield.test.js

```diff
@@ -131,6 +131,22 @@ test('set valid updates the component styles', () => {
   assert.isNotOk(root.classList.contains(cssClasses.INVALID));
 });
 
+test('set helptextValue updates the help text element content', () => {
+  const {component} = setupTest();
+  const helptext = getHelptext();
+  component.helptextElement = helptext;
+  component.helptextValue = 'foo';
+  assert.equal(helptext.textContent, 'foo');
+});
+
+test('get helptextValue retrieves the help text element content', () => {
+  const {component} = setupTest();
+  const helptext = getHelptext();
+  component.helptextElement = helptext;
+  helptext.textContent = 'foo';
```

Comment:
I don't think we need to set anything here first, right? The fixture has a value already.

Reply:
Ah, good find. I'll switch the check for the fixture's default value. Thanks!